superset/models/helpers.py

```diff
@@ -1314,7 +1315,33 @@ def get_time_filter(  # pylint: disable=too-many-arguments
             )
         return and_(*l)
 
-    def values_for_column(
+    def get_rls_filters_for_column(
```

Comment:
Seems this logic must exist somewhere else already, are we duplicating logic?

Reply:
Not using `sqlglot`. Ideally this logic would live in `superset/sql/parse.py`, since we're trying to centralize all the parsing and SQL manipulation logic there. But I'm in the process of converting the RLS logic implemented in `sqlparse` to use `sqlglot`, and I can do that.